modules/Bio/EnsEMBL/Variation/DBSQL/LDFeatureContainerAdaptor.pm

```diff
@@ -293,8 +299,8 @@ sub fetch_by_VariationFeature {
   $self->{_vf_name} = $vf->variation_name;
   
   # fetch by slice using expanded feature slice
-  my $max_snp_distance = $self->max_snp_distance || MAX_SNP_DISTANCE;
-  my $ldFeatureContainer = $self->fetch_by_Slice($vf->feature_Slice->expand($max_snp_distance, $max_snp_distance), $population);
+  my $max_snp_distance = $distance || $self->{max_snp_distance} || MAX_SNP_DISTANCE;
```

Comment:
This is very possibly a rebase/merge mistake on my part. I don't *think* I changed this deliberately, are you able to check through the history for me?

Reply:
[It](<https://github.com/willmclaren/ensembl-variation/commit/0a52927b73ba3d9f14c7e6b3c82bd7308981ceec#diff-454b511e2bf9922e332acdc1b57871d1>) does look like as if it was on purpose. But probably not needed because the distance is capped at 100kb. Nothing to worry about. I'll check the LD display on a sandbox using the backend code.